vms/vms/static/vms/js/load_city.js

```diff
@@ -9,8 +24,15 @@ $(document).ready(function() {
                 "country": countryId,
                 "state": stateId
             },
-            success: function(cities) {
-                $("#select_city").html(cities);
+            success: function(cities) { 
+		var state = $("#select_state");
+		var city = $("#select_city");
+		if(state.val() !== "0" && state.val()!==null && state.prop('disabled') === false) {
```

Comment:
Fix the indentation of this if/else block, we shouldn't have two tabs here. Sqaush the commits as well.